chat-server/server.js

```diff
@@ -14,13 +22,44 @@ const welcomeMessage = {
 
 //This array is our "data store".
 //We will start with one message in the array.
-//Note: messages will be lost when Glitch restarts our server.
 const messages = [welcomeMessage];
 
-app.get("/", function (request, response) {
+app.get("/", (request, response) => {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT,() => {
+app.get("/messages", (request, response) => {
+  response.json(messages);
+});
+app.get("/messages/:ID", (request, response) => {
+  const oneMess = request.params.ID;
```

Comment:
`oneMess` to me reads like it's a variable that's going to contain a message - does it? If not, is there a more clear name for this variable?